### PR TITLE
#238 Create a npm script to audit known vulnerabilities via yarn

### PR DESCRIPTION
**Pull request by konekoya**
### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #issue_number)
- [ ] ~Add unit tests (or please explain why this PR is not worth having new tests)~
- [x] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

required by #238

### CODE REVIEW

**Comment by konekoya** [comment]
This PR needs to wait for #459 to merge in

**Comment by vitojeng** [comment]
> This PR needs to wait for #459 to merge in

Should use `required by` in #459?

**Comment by konekoya** [comment]
> Should use required by in #459?

I'm not exactly sure about this... should I do that 😅

**Comment by konekoya** [comment]
@eechih @wu87988622 

After this PR gets merged in. Please upgrade your Yarn to `1.15.0` and run `yarn clean && yarn setup` again. Let's me know if there are any issues while you're upgrading/installing. Thanks!

File: docker/deps.dockerfile [hunk 1]
```diff
@@ -37,7 +37,7 @@ RUN curl -sL https://rpm.nodesource.com/setup_8.x | bash -
 RUN yum install -y nodejs
 
 # install yarn
-RUN npm install -g yarn@1.7.0
+RUN npm install -g yarn@1.15.0
```
Comment (konekoya) [inline docker/deps.dockerfile]: @chia7712 Does this look okay to you :)

File: docker/manager.dockerfile [hunk 2]
```diff
@@ -38,7 +38,7 @@ RUN curl -sL https://rpm.nodesource.com/setup_8.x | bash -
 RUN yum install -y nodejs
 
 # install yarn
-RUN npm install -g yarn@1.7.0
+RUN npm install -g yarn@1.15.0
```
Comment (konekoya) [inline docker/manager.dockerfile]: @chia7712  And here

File: docker/deps.dockerfile (hunk 1 above)
Comment (chia7712) [reply docker/deps.dockerfile]: Please test it on your local :)

File: docker/manager.dockerfile (hunk 2 above)
Comment (chia7712) [reply docker/manager.dockerfile]: Please test it on your local :)

**Comment by jackyoh** [comment]
retry

File: docker/deps.dockerfile (hunk 1 above)
Comment (konekoya) [reply docker/deps.dockerfile]: Will do, thanks :)

Comment (konekoya) [reply docker/deps.dockerfile]: Tested on my local, evertying seemed working just fined. Thanks for the help along the way :)

**Comment by konekoya** [comment]
@eechih @wu87988622 PR merged, please upgrade your yarn version and re-install your project deps again :) Thanks!

**Pull request merged**